department-of-veterans-affairs/abd-vro
# Update Ruby Version to become SecRel compliant

## What was the problem?
SecRel was complaining about non-compliancei n the bgs container because of internal ruby dependencies.
Associated tickets or Slack threads:
- #2258

## How does this fix it?[^1]
Updates Ruby version to 3.3.0

## How to test this PR
- run svc-bgs-api-integration-test github workflow and see it pass the integration tests and build 


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
